src/Extensions/Logging/Internal/EventSource/OmexLogEventSource.cs

```diff
@@ -8,7 +8,7 @@
 namespace Microsoft.Omex.Extensions.Logging
 {
 	// Renamed from Microsoft-OMEX-Logs to avoid conflict with sources in other libraries
-	[EventSource(Name = "Microsoft-OMEX-Logs-Ext")] //TODO: new event source should be registered GitHub Issue #187
+	[EventSource(Name = "Microsoft-OMEX-Logs-Ext")]
```

Comment:
>Microsoft-OMEX-Logs-Ext [](start = 22, length = 23)

is this the correct name?

Reply:
Yes, it's already in our configs, so I've removed TODO